Utilities/O2device/include/O2device/O2device.h

```diff
@@ -0,0 +1,67 @@
+/********************************************************************************
+ *    Copyright (C) 2014 Frankfurt Institute for Advanced Studies               *
+ *                                                                              *
+ *              This software is distributed under the terms of the             *
+ *         GNU Lesser General Public Licence version 3 (LGPL) version 3,        *
+ *                  copied verbatim in the file "LICENSE"                       *
+ ********************************************************************************/
```

Comment:
I think we have to conclude on a consistent copyright notice, some of the files have the notice copied from GSI, now we have some with FIAS, I vote for ame ALICE-phrase